index.html

```diff
@@ -10,229 +10,233 @@
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
-            </section>
-            
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Ruby on rails</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
+        <section id="about">
+            <div class="aboutme">
+                <h1>About me</h1>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hestiate to contact me.</p>
+                <a class="button" href="">Get My Resume</a>
+            </div>
+
+            <ul class="skills">
+                <li class="card">
+                    <img class="screenshot" src="/Images/languages-icon.svg" alt="languages">
+                    <h3>Languages</h3>
+                    <ul class="tags">
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                    </ul>  
+                </li>
+                <li class="card">
+                    <img class="screenshot" src="/Images/frameworks-icon.svg" alt="Frameworks">
+                    <h3>Frameworks</h3>
+                    <ul class="tags">
+                        <li>Bootstrap</li>
+                        <li>Ruby on rails</li>
+                        <li>RSpec</li>
+                        <li>Capybara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </li>
+                <li class="card">
+                    <img class="screenshot" src="/Images/skills-icon.svg" alt="Skills">
+                    <h3>Skills</h3>
+                    <ul class="tags">
+                        <li>Codekit</li>
+                        <li>GitHub</li>
+                        <li>codepen</li>
+                        <li>Gitlab</li>
+                        <li>Termial</li>
+                    </ul>
+                </li>
+            </ul>
         </section>
-
-    </section>
-
-
-    <section class="about">
-        <section class="aboutme">
-            <h1>About me</h1>
-            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don't hestiate to contact me.</p>
-             <a class="button" href="">Get My Resume</a>
+        <section id="form">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
+            <form 
```

Comment:
- [ ] As stated by the previous reviewer, please kindly reduce this big gap between the form and "About" sections in a way that matches the Figma design.
- [ ] As stated by the previous reviewer, please try and move the submit button so it's aligned to the left, as in the template.
- [ ] OPTIONAL: I strongly suggest you make the "textarea" field form bigger, like in the design, and make the other 2 narrower, like in the Figma.

Look:
![image](https://user-images.githubusercontent.com/57421823/149534144-e3c0ea3f-24fe-4ea2-b3c9-081fcdf3c0cd.png)